docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: (2015-10-29)
+- RFC PR: (leave this empty)
+- AMS Issue: (leave this empty)
+
+# Summary
+
+Provide a consistent API for the user of the AMS.
+
+# Motivation
+
+The actual public API is defined under `ActiveModelSerializers`,
+`ActiveModel::Serializer` and `ActiveModel`.
```

Comment:
So, that table is also part of https://github.com/rails-api/active_model_serializers/pull/1301 but I think also helps to explain the namespacing

`ActiveModel` comes from Rails.  Our lib is called `ActiveModelSerializers`  Rails has an `ActiveModel::Serializers::JSON` and `ActiveModel::Serialization` which reflect the history of the two libs, but in any case, I want AMS naming to be less confusing, and its interactions with Rails namespacing makes that harder, in my experience.

This is basically identifying the same naming issue as you and coming to the opposite decision.  The naming, as it stands, makes things more confusing, in my experience.

So, yes, I'm not changing the scope of the RFC, which is `AMS` vs. `AM::S`, but am elaborating on why `AMS` should remain the status quo.